priv/templates/nimble.phx.gen.template/.github/workflows/test.yml.eex

```diff
@@ -30,7 +32,11 @@ jobs:
         with:
           otp-version: ${{ env.OTP_VERSION }}
           elixir-version: ${{ env.ELIXIR_VERSION }}
-
+      <%= if web_project? do %>
+      - uses: actions/setup-node@v2-beta
```

Comment:
The stable version is [now released](https://github.com/actions/setup-node/releases/tag/v2.1.5):

```suggestion
      - uses: actions/setup-node@v2.1.5
```

Reply:
Thanks, I have fixed in https://github.com/nimblehq/elixir-templates/pull/79/commits/47035e61fc0877040fe376ca2312223eef025146